debian/copyright

```diff
@@ -1,28 +1,57 @@
-This package was debianized by Michael Prokop <mika@grml.org> on
-Thu, 28 Jul 2005 08:50:04 +0200.
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Source: https://github.com/grml/grml-hwinfo/
+
+Files: *
+Copyright: 2003-2025 Michael Prokop <mika@grml.org>
+License: GPL-2
+
+Files: grml-hwinfo
+License: GPL-2
```

Comment:
damn, here we really had GPL-2 only? :-/

Reply:
grml-hwinfo says:
> # License:       This file is licensed under the GPL v2.

i guess that means v2-only?